tests/test_bgbb.py

```diff
@@ -23,6 +23,15 @@
 N_CLIENTS_IN_SAMPLE = 10
 N_CLIENTS_ALL = 2 * N_CLIENTS_IN_SAMPLE
 
+first_dims = [
```

Comment:
An alternative to a CLI option is to have this as a module level constant that can be re-used in the test here.